src/frontend/components/ReceiptModal.tsx

```
@@ -0,0 +1,367 @@
+      {/* Hidden receipt content — React renders here, innerHTML is cloned to body for printing */}
+      <div style={{ position: 'absolute', left: '-9999px', top: '-9999px' }}>
+        <div id="wepos-print-receipt">
+          {proReceiptContent ? (
+            proReceiptContent
+          ) : (
+            <div className="wepos-checkout-print-wrapper">
+              {/* Header from admin settings */}
+              {receiptHeader && (
+                <div
+                  className="header"
+                  dangerouslySetInnerHTML={{ __html: receiptHeader }}
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)